hydra_redis/classes_objects.py

```diff
@@ -1,7 +1,15 @@
 import urllib.request
 import json
+import logging
 from redisgraph import Node, Edge
+from urllib.error import URLError, HTTPError
 
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+class RequestError(Exception):
```

Comment:
@sandeepsajan0 What is this doing?

Reply:
No, it's fine.

Reply:
Can we merge it?

Reply:
![annotation](https://user-images.githubusercontent.com/9114274/43727666-3d9cd174-99c0-11e8-960a-d06785550135.png)

Getting this error @sandeepsajan0

Reply:
Are you trying the update code? Because It is running fine with me.
![no_error](https://user-images.githubusercontent.com/23638629/43728089-9a133c62-99c1-11e8-9388-d37293731ac3.png)

And also I can try once run the setup.py file.

Reply:
It's probably a redisgraph issue. I think they've changed the structure a bit. Delete all docker imager and containers, create a new virtualenv and try again. You should see this error.

Reply:
Yeah, I got it!

Reply:
fixed